R/annotate_r_version.R

```diff
@@ -0,0 +1,29 @@
+#' Annotate R Version
+#'
+#' @importFrom rstudioapi getActiveDocumentContext insertText versionInfo
+#' @importFrom utils sessionInfo
+#'
+annotate_r_version <- function() {
+  context <- getActiveDocumentContext()
```

Comment:
perhaps we can add a message or a prompt if this function is called for an Rmd or Qmd file because adding the R version details at the top of a script will break the yaml preamble?